Problem-1.py

```diff
@@ -0,0 +1,61 @@
+from collections import deque
+#Time Complexity :O(1)
+#Space Complexity :O(n)
+#Did this code successfully run on Leetcode :yes
+#Any problem you faced while coding this :no
+class MyQueue(object):
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.inStack=deque()
+        self.outStack=deque()
```

Comment:
While the entire logic is correct, it would be better to use a stack since the question explicitly asks for it.